api/lambda_function.py

```diff
@@ -1,29 +1,103 @@
-# import the JSON utility package since we will be working with a JSON object
 import json
-# import the AWS SDK (for Python the package name is boto3)
 import boto3
-# import two packages to help us with dates and date formatting
 from time import gmtime, strftime
 
-# create a DynamoDB object using the AWS SDK
-dynamodb = boto3.resource('dynamodb')
-# use the DynamoDB object to select our table
-table = dynamodb.Table('Loyalty_Stamps')
-# store the current time in a human readable format in a variable
+dynamodb = boto3.resource('dynamodb', region_name='us-west-2')
+table = dynamodb.Table('StampInfo')
 now = strftime("%a, %d %b %Y %H:%M:%S +0000", gmtime())
 
-# define the handler function that the Lambda service will use an entry point
 def lambda_handler(event, context):
```

Comment:
whats parameter <context> ?

Reply:
It is just an object AWS passes in which holds metadata about the environment and the caller. It's not really useful for us 
https://docs.aws.amazon.com/lambda/latest/dg/python-context.html